runtime/util_core/vmargs_core.c

```diff
@@ -360,11 +370,11 @@ parseOptionsBuffer(J9PortLibrary* portLib, char* argumentBuffer, J9JavaVMArgInfo
 		if (tokenEnd) {
 			if ((0 == strncmp(tokenStart, VMOPT_XOPTIONSFILE_EQUALS, strlen(VMOPT_XOPTIONSFILE_EQUALS))) && (TRUE == parseOptionsFileFlag)) {
 				/* addXOptionsFile allocates a new buffer for the -Xoptionsfile argument and the file contents */
-				if (0 != addXOptionsFile(portLib, tokenStart, vmArgumentsList, verboseFlags)) {
+				if (0 != addXOptionsFile(portLib, tokenStart, vmArgumentsList, verboseFlags, TRUE)) {
 					return -1;
 				}
 			} else {
-				J9JavaVMArgInfo *optArg = newJavaVMArgInfo(vmArgumentsList, NULL, CONSUMABLE_ARG);
+				optArg = newJavaVMArgInfo(vmArgumentsList, NULL, CONSUMABLE_ARG | (parseEnvVar)? USER_ARG : 0);
```

Comment:
Nitpick: parseEnvVar doesn't need the brackets. Also there should be a space before the `?`